test/e2e/scaffold/httpbin.go

```diff
@@ -84,7 +86,8 @@ spec:
 )
 
 func (s *Scaffold) newHTTPBIN() (*corev1.Service, error) {
-	if err := k8s.KubectlApplyFromStringE(s.t, s.kubectlOptions, _httpbinDeployment); err != nil {
+	httpbinDeployment := fmt.Sprintf(_httpbinDeployment, 1)
```

Comment:
If treat the _httpbinDeployment as a template, we should change its name to `_httpbinDeploymentTemplate`.

Reply:
Done